webcompat/helpers.py

```diff
@@ -716,3 +716,7 @@ def ab_init(response):
             response.set_cookie(exp_id, var, max_age=max_age)
 
     return response
+
+
+# Register `ab_active` as jinja function
+app.jinja_env.globals.update(ab_active=ab_active)
```

Comment:
I'm not a big fan of this hanging out in the bottom of a module function-less. That seems not at the right place. Was there a reason why it was not up there with cache_dict and other things.

Reply:
@johngian can we follow the way we do this for `bust_cache`? It seems like it's equivalent, and would be nice to be consistent.

https://flask.palletsprojects.com/en/1.1.x/templating/#registering-filters

Reply:
Sure thing. I just followed the flask documentation and looked like a pretty common way to configure global functions.